rules/rbe_repo.bzl

```diff
@@ -18,24 +18,63 @@ The toolchain configs (+ platform) produced/selected by this rule can be used
 to, e.g., run a remote build in which remote actions will run inside a
 container image.
 
-Exposes the rbe_autoconfig macro that does the following:
-- If users selects the standard rbe-ubuntu 16_04 image, create aliases to
-  the appropriate toolchain / platform targets for the current version of Bazel
-- Otherwise, pull the selected toolchain container image (using 'docker pull').
-- Starts up a container using the pulled image, mounting either a small sample
+Exposes the rbe_autoconfig macro that encapsulates all functionality to
+create and use toolchain configs. The main use cases for this rule are
+  1. If you use the rbe-ubuntu 16_04 image in your RBE builds: This macro
```

Comment:
Suggest pointing to a doc or the GCR link for the RBE Ubuntu 16.04 image for completeness

Reply:
done